src/Datadog.Trace/Logging/FallbackNLogLogProvider.cs

```diff
@@ -0,0 +1,73 @@
+// <copyright file="FallbackNLogLogProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq.Expressions;
+using Datadog.Trace.Logging.LogProviders;
+
+namespace Datadog.Trace.Logging
+{
+    /// <summary>
+    /// Log provider for all versions of NLog lower than 4.1. This is required
```

Comment:
Might be nice to clarify that this provider _supports_ all versions of NLog lower than 4.1, but it will only be _used_ with NLog version 1.x?

Reply:
Added more documentation in my latest commit. And it also clarifies that this is actually being used for all versions of NLog lower than 4.1. So it's the `CustomNLogLogProvider` for the most optimized performance in NLog >= 4.1 and `FallbackNLogLogProvider` for most compatible fallback for NLog < 4.1 😄